docs/building-a-form/available-form-features-and-usage-guidelines.md

```diff
@@ -419,14 +419,29 @@ For the code implementation, see the [`review` folder](../../src/js/review).
 
 ### Required checkbox before form submission
 
-Use this feature to require a user to indicate they have read terms and conditions, a privacy policy, or any other text before submitting your form. It includes a checkbox and short-form text that can include relevant links to more verbose information on separate pages on your site. To configure this feature, place a `preSubmitInfo` object in the `formConfig`:
+Use this feature to require a user to indicate they have read terms and conditions, a privacy policy, or any other text before submitting your form. It includes a checkbox and short-form text that can include relevant links to more verbose information on separate pages on your site.
+
+To configure this feature, place a `preSubmitInfo` object in the `formConfig`. These are the available options:
+<dl>
+<dt><code>notice</code></dt>
+<dd>A text string or <a href="https://reactjs.org/docs/rendering-elements.html">React element</a> that is placed above the checkbox and submit button. If the form definition file is `.jsx` the definition can be inline, or use `import` to reference an external component. Optional; if nothing is given for this property no notice appears.</dd>
```

Comment:
> Optional; if nothing is given for this property no notice appears.

It's a good idea to state whether the properties are optional up front, like so:

"`notice` (Optional): A text string or React element that..."